trillian/ctfe/instance.go

```diff
@@ -177,3 +206,25 @@ func ValidateLogMultiConfig(cfg *configpb.LogMultiConfig) (map[string]*configpb.
 
 	return backendMap, nil
 }
+
+// ToMultiLogConfig creates a multi backend config proto from the data
+// loaded from a single backend configuration file. All the log configs
+// reference a default backend spec as provided.
+func ToMultiLogConfig(cfg []*configpb.LogConfig, beSpec string) (*configpb.LogMultiConfig, map[string]*configpb.LogBackend) {
+	defaultName := "default"
```

Comment:
I think this could be clearer with inline initializations, something like:
```go
func ToMultiLogConfig(cfg []*configpb.LogConfig, beSpec string) (*configpb.LogMultiConfig, map[string]*configpb.LogBackend) {
	defaultBackend := &configpb.LogBackend{Name: "default", BackendSpec: beSpec}
	for _, c := range cfg {
		c.LogBackendName = defaultBackend.Name
	}
	return &configpb.LogMultiConfig{
		LogConfigs: &configpb.LogConfigSet{Config: cfg},
		Backends:   &configpb.LogBackendSet{Backend: []*configpb.LogBackend{defaultBackend}},
	}, map[string]*configpb.LogBackend{defaultBackend.Name: defaultBackend}
}
```

Reply:
Done.